Contentless/Program.cs

```diff
@@ -43,11 +44,62 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToList().ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
+        if (config.References.Any())
```

Comment:
- This is an array, so please use `Count`, which is much faster
- Why create `referencesVersions` first and only *then* check whether there even are any references?

Reply:
`Any` I fixed,  block after this need `referencesVersions `

Reply:
I change two blocks: firstly, for parsing versions and , secondly, for changing` .mgcb`.